pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -527,10 +529,14 @@ func (rc *reconciler) reconstructVolume(volume podVolume) (*reconstructedVolume,
 	}
 
 	reconstructedVolume := &reconstructedVolume{
-		volumeName:          uniqueVolumeName,
-		podName:             volume.podName,
-		volumeSpec:          volumeSpec,
-		outerVolumeSpecName: volumeName, /* volumeName is InnerVolumeSpecName. But this information will not be used for cleanup */
+		volumeName: uniqueVolumeName,
```

Comment:
There is an else to set the volume name

} else {
 		uniqueVolumeName = volumehelper.GetUniqueVolumeNameForNonAttachableVolume(volume.podName, plugin, volumeSpec)		 		uniqueVolumeName = volumehelper.GetUniqueVolumeNameForNonAttachableVolume(volume.podName, plugin, volumeSpec)
 	}		 	}